webpack/ForemanInventoryUpload/Components/TabHeader/TabHeader.js

```diff
@@ -1,26 +1,57 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Grid, Button, Icon } from 'patternfly-react';
+import { Grid, Button, Icon, DropdownItem, Dropdown, DropdownToggle, DropdownToggleAction } from 'patternfly-react';
 import { noop } from 'foremanReact/common/helpers';
 import { sprintf, translate as __ } from 'foremanReact/common/I18n';
 import { isExitCodeLoading } from '../../ForemanInventoryHelpers';
 import './tabHeader.scss';
 
-const TabHeader = ({ exitCode, onRestart, onDownload, toggleFullScreen }) => (
+const onActionToggle = () => {
+  setIsActionOpen(prev => !prev);
+};
+
+const dropdownItems = [
+  <DropdownItem
+    aria-label="restart_disconnected"
+    ouiaId="restart_disconnected"
+    key="restart_disconnected"
+    component="button"
+    onClick={restartDisconnected}
```

Comment:
in case you would still like to use Redux in this PR,
the prop `restartDisconnected` would be passed to the `TabHeader` component, maybe you want to pass the `dropdownItems` into that component, e.g:

```js
const TabHeader = ({ exitCode, onRestart, onDownload, restartDisconnected, toggleFullScreen }) => {
const dropdownItems = [....]
return <Grid.Row className="tab-header">.......</Grid.Row>
}
```

Reply:
When I put in `const dropdownItems = [....]` everything lights up red like a stop light :rofl: the error from vscode says 

```javascript
Identifier expected. 'const' is a reserved word that cannot be used here.ts(1359)
```